docs/installation/mailing.md

```diff
@@ -2,10 +2,18 @@
 
 Autolab requires mailing to allow users to register accounts and reset passwords. You will also be able to make announcements through Autolab as well. The recommended approach is to setup Autolab to use a SMTP server, such as [mailgun](https://mailgun.com), [SendGrid](https://sendgrid.com), [Amazon SES](https://aws.amazon.com/ses/) or any other valid SMTP mail servers to send out email.
 
-We intend this instructions mainly for production usage. 
```

Comment:
You may want to add a note / footnote about how now we only use the UI config and that env file based config has been removed as a disclaimer.

Reply:
Oh okay I see that all of the old instructions are preserved, but maybe make a heading for v3.0.0 so it's clear that for newer versions they should use the UI config and after and add a screenshot or something of the page?

Reply:
Made a heading.